packages/account/src/Components/financial-details/financial-details-partials.tsx

```diff
@@ -27,39 +32,45 @@ type TFinancialDetailsDropdownFieldProps = {
  * @param {string} field_key - field reference of the field
  * @param {string} placeholder - placeholder of the field
  * @param {string} label - label of the field
+ *  @param {string} employment_status - selected employment_status,
  * @returns {JSX.Element}
  */
 const FinancialDetailsDropdownField = ({
     dropdown_list,
     field_key,
     placeholder = localize('Please select'),
     label,
+    employment_status,
 }: TFinancialDetailsDropdownFieldProps) => {
     const { values, handleChange, handleBlur, touched, errors, setFieldValue } = useFormikContext<{
         [key: string]: string;
     }>();
 
+    const getFormattedOccupationValues = () =>
+        employment_status === 'Employed' && values?.occupation === 'Unemployed' ? '' : values?.occupation;
```

Comment:
How does this being handled in different languages? 
Isn't it better not to hard code the values?